Template/LO/SubProcesses/setscales.f

```diff
@@ -108,6 +109,7 @@ subroutine set_fac_scale(P,q2factorization)
 c
       include 'genps.inc'
       include 'nexternal.inc'
+      include 'vector.inc'
       include 'coupl.inc'
```

Comment:
same

Reply:
same as above, vector.inc is needed by coupl.inc